[vulkan] Remove unused variables

IvanKobzarev opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#39410 [vulkan] Remove unused variables**
* #39384 [vulkan] Shaders caching
* #39282 [vulkan] jit passes for vulkan conv2 prepack and fuse with clamp

Differential Revision: [D21962429](https://our.internmc.facebook.com/intern/diff/D21962429)

IvanKobzarev committed:
[vulkan] Remove unused variables

[ghstack-poisoned]

IvanKobzarev committed:
Update on "[vulkan] Remove unused variables"




[ghstack-poisoned]

IvanKobzarev committed:
Update on "[vulkan] Remove unused variables"


Differential Revision: [D21962429](https://our.internmc.facebook.com/intern/diff/D21962429)

[ghstack-poisoned]

IvanKobzarev committed:
Update on "[vulkan] Remove unused variables"


Differential Revision: [D21962429](https://our.internmc.facebook.com/intern/diff/D21962429)

[ghstack-poisoned]

IvanKobzarev committed:
Update on "[vulkan] Remove unused variables"


Differential Revision: [D21962429](https://our.internmc.facebook.com/intern/diff/D21962429)

[ghstack-poisoned]

IvanKobzarev committed:
Update on "[vulkan] Remove unused variables"


Differential Revision: [D21962429](https://our.internmc.facebook.com/intern/diff/D21962429)

[ghstack-poisoned]

IvanKobzarev committed:
Update on "[vulkan] Remove unused variables"


Differential Revision: [D21962429](https://our.internmc.facebook.com/intern/diff/D21962429)

[ghstack-poisoned]

IvanKobzarev committed:
Update on "[vulkan] Remove unused variables"


Differential Revision: [D21962429](https://our.internmc.facebook.com/intern/diff/D21962429)

[ghstack-poisoned]

IvanKobzarev committed:
Update on "[vulkan] Remove unused variables"


Differential Revision: [D21962429](https://our.internmc.facebook.com/intern/diff/D21962429)

[ghstack-poisoned]

IvanKobzarev committed:
Update on "[vulkan] Remove unused variables"


Differential Revision: [D21962429](https://our.internmc.facebook.com/intern/diff/D21962429)

[ghstack-poisoned]

IvanKobzarev committed:
Update on "[vulkan] Remove unused variables"


Differential Revision: [D21962429](https://our.internmc.facebook.com/intern/diff/D21962429)

[ghstack-poisoned]

IvanKobzarev committed:
Update on "[vulkan] Remove unused variables"


Differential Revision: [D21962429](https://our.internmc.facebook.com/intern/diff/D21962429)

[ghstack-poisoned]

IvanKobzarev committed:
Update on "[vulkan] Remove unused variables"


Differential Revision: [D21962429](https://our.internmc.facebook.com/intern/diff/D21962429)

[ghstack-poisoned]

IvanKobzarev committed:
Update on "[vulkan] Remove unused variables"


Differential Revision: [D21962429](https://our.internmc.facebook.com/intern/diff/D21962429)

[ghstack-poisoned]

IvanKobzarev committed:
Update on "[vulkan] Remove unused variables"


Differential Revision: [D21962429](https://our.internmc.facebook.com/intern/diff/D21962429)

[ghstack-poisoned]

IvanKobzarev committed:
Update on "[vulkan] Remove unused variables"


Differential Revision: [D21962429](https://our.internmc.facebook.com/intern/diff/D21962429)

[ghstack-poisoned]

AshkanAliabadi reviewed (approved):
Great.  This could be improved with better const correctness.

IvanKobzarev committed:
Update on "[vulkan] Remove unused variables"


Differential Revision: [D21962429](https://our.internmc.facebook.com/intern/diff/D21962429)

[ghstack-poisoned]

IvanKobzarev committed:
Update on "[vulkan] Remove unused variables"


Differential Revision: [D21962429](https://our.internmc.facebook.com/intern/diff/D21962429)

[ghstack-poisoned]

IvanKobzarev committed:
Update on "[vulkan] Remove unused variables"


Differential Revision: [D21962429](https://our.internmc.facebook.com/intern/diff/D21962429)

[ghstack-poisoned]

IvanKobzarev committed:
Update on "[vulkan] Remove unused variables"


Differential Revision: [D21962429](https://our.internmc.facebook.com/intern/diff/D21962429)

[ghstack-poisoned]

IvanKobzarev committed:
Update on "[vulkan] Remove unused variables"


Differential Revision: [D21962429](https://our.internmc.facebook.com/intern/diff/D21962429)

[ghstack-poisoned]

IvanKobzarev committed:
Update on "[vulkan] Remove unused variables"


Differential Revision: [D21962429](https://our.internmc.facebook.com/intern/diff/D21962429)

[ghstack-poisoned]

IvanKobzarev committed:
Update on "[vulkan] Remove unused variables"


Differential Revision: [D21962429](https://our.internmc.facebook.com/intern/diff/D21962429)

[ghstack-poisoned]

IvanKobzarev committed:
Update on "[vulkan] Remove unused variables"


Differential Revision: [D21962429](https://our.internmc.facebook.com/intern/diff/D21962429)

[ghstack-poisoned]

IvanKobzarev committed:
Update on "[vulkan] Remove unused variables"


Differential Revision: [D21962429](https://our.internmc.facebook.com/intern/diff/D21962429)

[ghstack-poisoned]

IvanKobzarev committed:
Update on "[vulkan] Remove unused variables"


Differential Revision: [D21962429](https://our.internmc.facebook.com/intern/diff/D21962429)

[ghstack-poisoned]

IvanKobzarev committed:
Update on "[vulkan] Remove unused variables"


Differential Revision: [D21962429](https://our.internmc.facebook.com/intern/diff/D21962429)

[ghstack-poisoned]
